src/renderer/Core/getImageUrl.ts

```diff
@@ -1,7 +1,5 @@
 import type { Image } from "@common/Core/Image";
 
 export const getImageUrl = ({ image, shouldPreferDarkColors }: { image: Image; shouldPreferDarkColors: boolean }) => {
-    return shouldPreferDarkColors
-        ? (image.urlOnDarkBackground ?? image.url)
-        : (image.urlOnLightBackground ?? image.url);
+    return shouldPreferDarkColors ? image.urlOnDarkBackground ?? image.url : image.urlOnLightBackground ?? image.url;
```

Comment:
?

Reply:
I ran the lint command and this was changed

Reply:
Make sure to update this branch and reinstall the dependencies with `npm install`. Then this should not be changed.